README.md

```diff
@@ -1,27 +1,188 @@
-# Role Name
+# Mattermost
 
-A brief description of the role goes here.
+This role sets up and configures a mattermost instance.
 
 
 ## Requirements
 
-Any pre-requisites that may not be covered by Ansible itself or the role should be mentioned here.
-For instance, if the role uses the EC2 module or depends on other Ansible roles, it may be a good idea to mention in this section that the boto package is required.
-
+It needs an apt based system like ubuntu. Also the stuvusIT.nginx and stuvusIT.postgresql roles are required.
```

Comment:
Capitalize Ubuntu. Also add links to the two roles. And is nginx required or optional? Seems optional to me

Reply:
fixed